src/main/java/org/jabref/logic/formatter/bibtexfields/RemoveHyphenatedNewlinesFormatter.java

```diff
@@ -10,7 +10,8 @@
  * Removes all hyphenated line breaks in the string.
  */
 public class RemoveHyphenatedNewlinesFormatter extends Formatter {
-    private static final Pattern HYPHENATED_WORDS = Pattern.compile("(-\r\n|-\n|-\r)");
+    private static final String newLine = String.format("%n");
+    private static final Pattern HYPHENATED_WORDS = Pattern.compile("(-" + newLine + ")");
```

Comment:
You can also use `\R` to match any line breaks, see https://docs.oracle.com/javase/9/docs/api/java/util/regex/Pattern.html.

Reply:
Thanks, it is what I didn't know before. But it seems `\R` will match any of `\r`, `\n` and `\r\n`.
![image](https://user-images.githubusercontent.com/24741764/43588283-bdeaa01a-969e-11e8-9c47-b9a5b27ebb1f.png)
In this example, you can see `\r` is matched on Windows. But we need only match `\r\n` on Windows, so I think `%n` is the right choice.